config/mock.go

```diff
@@ -75,13 +77,17 @@ type MockConfig struct {
 	DryRun                           bool
 	DryRunFieldName                  string
 	AddHostMetadataToTrace           bool
```

Comment:
If we choose to use `AddHostMetadataToTraceVal`, we probably should delete `AddHostMetadataToTrace`.

Reply:
That's fair. I was trying to limit my changes a bit so I introduced some duplication here, but if I get the build passing I will try to not duplicate these values.

Reply:
Resolved this with the build fix. Thanks for the help!